test/notifications/display_test.dart

```diff
@@ -1256,39 +1258,39 @@ void main() {
     });
   });
 
-  group('NotificationOpenPayload', () {
+  group('NotificationNavigationData', () {
```

Comment:
Since the code this is testing is getting moved between files, the tests should move to remain in a corresponding file.